modules/moordyn/src/MoorDyn_Rod.f90

```diff
@@ -951,20 +950,20 @@ SUBROUTINE Rod_DoRHS(Rod, m, p)
       ! >>> some of the kinematics parts of this could potentially be moved to a different routine <<<
       Rod%OrMat = CalcOrientation(phi, beta, 0.0_DbKi)        ! get rotation matrix to put things in global rather than rod-axis orientations
       
-      Imat = RotateM3(Imat_l, Rod%OrMat)  ! rotate to give inertia matrix about CG in global frame
+      Rod%Imat = RotateM3(Imat_l, Rod%OrMat)  ! rotate to give inertia matrix about CG in global frame
       
       ! these supplementary inertias can then be added the matrix (these are the terms ASIDE from the parallel axis terms)
-      Rod%M6net(4:6,4:6) = Rod%M6net(4:6,4:6) + Imat
+      Rod%M6net(4:6,4:6) = Rod%M6net(4:6,4:6) + Rod%Imat
       
 
       ! now add centripetal and gyroscopic forces/moments, and that should be everything
       h_c = 0.5*Rod%UnstrLen          ! distance to center of mass
       r_c = h_c*Rod%q                 ! vector to center of mass
       
       ! note that Rod%v6(4:6) is the rotational velocity vector, omega   
-      Fcentripetal = 0.0_DbKi !<<<TEMP<<< -cross_product(Rod%v6(4:6), cross_product(Rod%v6(4:6), r_c ))*Rod%mass <<<
-      Mcentripetal = 0.0_DbKi !<<<TEMP<<< cross_product(r_c, Fcentripetal) - cross_product(Rod%v6(4:6), MATMUL(Imat,Rod%v6(4:6)))
-      
+      Fcentripetal = -cross_product(Rod%v6(4:6), cross_product(Rod%v6(4:6), r_c ))*Rod%mass
+      Mcentripetal = cross_product(r_c, Fcentripetal) - cross_product(Rod%v6(4:6), MATMUL(Rod%Imat,Rod%v6(4:6)))
```

Comment:
Should we leave a comment here explaining the extra `cross_product(r_c, Fcentripetal)` term coming from using the moment of inertia about the CoG? Just in case we forget again in the future.

Reply:
Resolved with https://github.com/OpenFAST/openfast/pull/2334/commits/f164d946650f2207a0476cc697b542467c175c72